internal/server/server.go

```diff
@@ -10,20 +10,21 @@ import (
 	"google.golang.org/grpc"
 
 	"github.com/kubernetes-csi/csi-proxy/client"
+	srvtypes "github.com/kubernetes-csi/csi-proxy/internal/server/types"
```

Comment:
Need to amend the auto-gen to account for this, will do as this PR gets merged.

Reply:
Thanks! Note that the auto-gen will also need to handle https://github.com/kubernetes-csi/csi-proxy/pull/11/files#diff-d81281b59794bf1f85ac717a4410af56.